modules/libcom/test/epicsThreadPoolTest.c

```diff
@@ -8,7 +8,7 @@
 #include "epicsThreadPool.h"
 
 /* included to allow tests to peek */
-#include "../../src/pool/poolPriv.h"
+#include "../src/pool/poolPriv.h"
```

Comment:
Sorry, this change is wrong when using the EPICS build system. When the epicsThreadPoolTest.c file is compiled the working directory is always `modules/libcom/test/O.$(T_A)` so the relative path to `modules/libcom/src` must be `../../src`.